Language/Apache.php

```diff
@@ -71,7 +72,8 @@ public function setupRules()
             ]),
             'operator.punctuation' => new Rule(new RegexMatcher('/([\(\){},;])/i')),
             'keyword.format' => new Rule(
-                new RegexMatcher('/(%[\w%][-+#0]?(?:[0-9]+|\*)?(?:\.(?:[0-9]+|\*))?)/'), [
+                new RegexMatcher('/(%[\w%][-+#0]?(?:[0-9]+|\*)?(?:\.(?:[0-9]+|\*))?)/'),
```

Comment:
I prefer to have opening bracket on the same line - it's easier to read in my opinion because we don't have blank space between lines and they visually appear more bounded.